Tasks/VsTest/vstest.ts

```diff
@@ -187,6 +187,28 @@ function generateResponseFile(): Q.Promise<string> {
     return defer.promise;
 }
 
+function publishCodeChanges(): Q.Promise<string> {
+    var defer = Q.defer<string>();
+    var selectortool = tl.createToolRunner(tiaSelectorTool);
+    selectortool.arg("PublishCodeChanges");
+    selectortool.arg("/TfsTeamProjectCollection:" + tl.getVariable("System.TeamFoundationCollectionUri"));
+    selectortool.arg("/ProjectId:" + tl.getVariable("System.TeamProject"));
+    selectortool.arg("/newdropPath:" + tl.getVariable("Build.SourcesDirectory"));
+    selectortool.arg("/Definitionid:" + tl.getVariable("System.DefinitionId"));
+    selectortool.arg("/buildid:" + tl.getVariable("Build.BuildId"));
+    selectortool.arg("/token:" + tl.getVariable("System.AccessToken"));
```

Comment:
Availability of tiaSelectorTool will be taken care by isTiaAllowed func which will check for existence tiaSelectorTool before invoking any tia related functions